cases/src/org/commcare/cases/util/CasePurgeFilter.java

```diff
@@ -315,6 +324,23 @@ public boolean invalidEdgesWereRemoved() {
         return this.invalidEdgesWereRemoved;
     }
 
+    public String getMissingCasesString() {
+        return flattenVectorOfStrings(this.missingCases);
+    }
+
+    public String getRemovedCasesString() {
+        return flattenVectorOfStrings(this.casesRemovedDueToMissingCases);
+    }
+
+    private static String flattenVectorOfStrings(Vector<String> v) {
+        String s = "";
+        for (String caseId : v) {
+            s = s + caseId + ", ";
+        }
+        s = s.substring(0, s.length()-2);
```

Comment:
This will throw an exception if there are no strings in the vector. I would honestly just space separate them and not worry about the hanging space at the end.